pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json

```diff
@@ -605,6 +615,18 @@
     "any",
     "base"
   ],
+  "fedora.23-arm64": [
+    "fedora.23-arm64",
```

Comment:
On the one hand, it looks like Fedora 23 didn't support arm64/aarch64 (lack of an F23 in https://fedoraproject.org/wiki/Category:AArch64).  On the other hand, it probably doesn't matter, and isn't worth splitting the definition in the props file over this.

Reply:
If know a clean way to split versions, I would be happy to make this conditional on, say, Fedora 30 and later.